frontend/src/app/components/homepage/homepage.component.html

```diff
@@ -0,0 +1,14 @@
+<table border="1" style="border-collapse: collapse;">
```

Comment:
Moim zdaniem pisanie styli w ten sposób jest nieprawidłowe. Stwórz nową klasę i w niej dodaj te style. Łatwiej to wtedy rozwijać i w łatwiej jest się w tym wszystkim połapać, bo jak częśc będziesz pisał w tej postaci a potem coś dodasz do pliku css to możesz być bardzo zaskoczony tym co się pojawi na ekranie.